static/src/js/providers/AppContextProvider/AppContextProvider.jsx

```diff
@@ -32,35 +36,92 @@ const AppContextProvider = ({ children }) => {
   const [draft, setDraft] = useState()
   const [savedDraft, setSavedDraft] = useState()
   const [user, setUser] = useState({})
+  const [providerIds, setProviderIds] = useState([]) // State to hold providerIds
+
+  const { addNotification } = useNotificationsContext() || {}
 
   const { keywords } = keywordsContext
 
   useEffect(() => {
     setUser({
       name: 'User Name',
-      token: 'ABC-1',
-      providerId: 'MMT_2'
+      token: 'ABC-1'
     })
   }, [])
 
   const login = useCallback(() => {
     setUser({
       name: 'User Name',
-      token: 'ABC-1',
-      providerId: 'MMT_2'
+      token: 'ABC-1'
     })
   })
 
   const logout = useCallback(() => {
     setUser({})
   })
 
+  const setProviderId = useCallback((providerId) => {
+    setUser((prevUser) => ({
+      ...prevUser,
+      providerId
+    }))
+  }, [])
+
+  // Fetch providers using useQuery hook
+  const [getProvider] = useLazyQuery(GET_ACLS, {
+    variables: {
```

Comment:
These lines are not covered. I see I your test you are mocking the `LazyQuery`. Instead of mocking the @apollo/client you need to mock actual graphql query. Look at MetadataForm.test.js as example and see how getDraft is being mocked.